src/guides/v2.4/graphql/mutations/set-shipping-address.md

```diff
@@ -6,7 +6,7 @@ title: setShippingAddressesOnCart mutation
 The `setShippingAddressesOnCart` mutation sets one or more shipping addresses on a specific cart. The shipping address does not need to be specified in the following circumstances:
 
 *  The cart contains only virtual items
-*  When you defined the billing address, you set the `same_for_shipping` attribute to `true`. Magento assigns the same address as the shipping address.
```

Comment:
Should this not also be updated on -
src/guides/v2.3/graphql/mutations/set-shipping-address.md

Reply:
@BarnyShergold, Yes, it should also change on version 2.3 page as well.

Reply:
The 2.4 version of this file is not symlinked to the 2.3 version, so you will need to change the 2.3 file. If you can do this then I can approve the change.

Reply:
Hello @BarnyShergold , Sure, I will do that. Just one question. Do I need to create pull request for that separately or can I add that change into this pull request only? can you please suggest me the correct way as this is my first pull request only.

Reply:
Same pull request please - then both get linked. Just make the change to 2.3, commit & push.

I'll get an email when you have done it

Reply:
@pratiknavapara @BarnyShergold The symlink between the 2.4 and 2.3 versions of the topic because the content has changed. The exact same change must be made in the 2.3 version, in this PR.

Reply:
@keharper - Yes I've already requested the author to do this.

Reply:
Hello @BarnyShergold , I have created a pull request for the same change for version 2.3 https://github.com/magento/devdocs/pull/7812. I don't have know how to link it with this existing pull request. Can you please help me out?